### DISCUSSION TITLE
Hexo美化|Cards主题 - 没有秃头基因的燊的博客

### DISCUSSION THREAD

**Discussion by giscus** [yzs020220/giscus-repo#4] [Announcements]
# Hexo美化|Cards主题 - 没有秃头基因的燊的博客

记录了刚开始这个博客怎么美化主题的，希望后来人能少走弯路吧，遇到问题多看官方文档

https://yzs020220.github.io/posts/22490/

**Comment by HCY-ASLEEP** [comment]
666
